src/types/TypeError.tsx

```diff
@@ -0,0 +1,5 @@
+export enum TypeError {
+  DEFAULT = '',
+  TITLE_EERROR = 'Please enter a title',
```

Comment:
[CODE STYLE] - The enum member `TITLE_EERROR` seems to have a typo. It should be `TITLE_ERROR`. Enum members should be correctly spelled to avoid confusion and potential bugs in the code.